modules/ob-examples-of-tekton-resources-management.adoc

```diff
@@ -0,0 +1,146 @@
+// This module is included in the following assembly:
+//
+// builds/configuring-build-strategies.adoc
+
+:_content-type: CONCEPT
+[id="ob-examples-of-tekton-resources-management_{context}"]
+= Examples of Tekton resources management
+
+The following examples show how resource requests are managed by Tekton based on whether the `LimitRange` parameter is defined in that namespace.
+
+.Example: Namespace without the `LimitRange` parameter, both steps with the same resource values
```

Comment:
I think that this could be its own section + procedure

Reply:
@stevsmit I would call this a conceptual stuff because a user does not need to perform any procedure in context of Tekton resource management. 
This conceptual module basically focuses on how Tekton performs resource allocation for a build step based on the values a user defined in a build startegy CR (the same is explained in different scenario-based examples). The configuration examples for defining build step resources has already been discussed in the `defining step resources` section. 
This partcular section `Resource management in tekton Pipelines` actually highlights the role of Tekton in resource allocation based on the values a user defined in a build strategy. After going through this module, a user can build an understanding around defining recource limit for a particular build step.